include/rcutils/logging.h

```diff
@@ -56,14 +117,15 @@ typedef struct rcutils_log_location_t
   size_t line_number;
 } rcutils_log_location_t;
 
-/// The severity levels of log message.
+/// The severity levels of log messages / logger thresholds.
 enum RCUTILS_LOG_SEVERITY
 {
   RCUTILS_LOG_SEVERITY_DEBUG = 0,  ///< The debug log level
   RCUTILS_LOG_SEVERITY_INFO = 1,  ///< The info log level
   RCUTILS_LOG_SEVERITY_WARN = 2,  ///< The warn log level
   RCUTILS_LOG_SEVERITY_ERROR = 3,  ///< The error log level
   RCUTILS_LOG_SEVERITY_FATAL = 4,  ///< The fatal log level
+  RCUTILS_LOG_SEVERITY_UNSET = 100,  ///< The unset log level
```

Comment:
API not specifically checking for this would consider it to be higher than fatal which would hide all further log messages. Therefore I would prefer the unset value to be *lower* than all real severity values.

Reply:
OK, I will move it to be 0 and bump the other levels higher. I will copy python's logger levels https://docs.python.org/3/library/logging.html#logging-levels and set them with gaps of 10, and update rclpy to match.

Reply:
bf54dd61ef30dc11caaeaceeb8008c3f6de29148